Src/VimCore/CoreInterfaces.fs

```diff
@@ -4664,6 +4664,9 @@ type IVimHost =
     /// Is the ITextBuffer in a dirty state?
     abstract IsDirty: textBuffer: ITextBuffer -> bool
 
+    /// Whether the ITextView is loaded

```

Comment:
Given the use case here think we should add something to the comment that indicates the view is ready for operations to be performed on it.

Reply:
I'll do that. Now that I'm thinking about it, I wonder whether it should be named literally. The entire reason we need host support is that we need information that only the IWpfTextView can give us and we don't have access to WPF from Vim.Core by design. Hmm, I think the host should provide the DoWhenReady functionality and then we no longer need IsLoaded.